joblib/numpy_pickle.py

```diff
@@ -89,7 +89,7 @@ def write_array(self, array, pickler):
                                                   'zerosize_ok'],
                                            buffersize=buffersize,
                                            order=self.order):
-                pickler.file_handle.write(chunk.tostring('C'))
+                pickler.file_handle.write(chunk.data)
```

Comment:
From what I remember this code was taken straight from numpy. Maybe it is worth creating a numpy PR.